testv1.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source common.sh
+
+function test_init_project {
+  header_text "performing init project"
+  kubebuilder init --project-version=v1 --domain example.com <<< "y"
+}
```

Comment:
Don't we want to run `make` here like we are doing in `test_init_project_manual_dep_ensure` ?

Reply:
Added